### DISCUSSION TITLE
Where is the collected forms in Contact Us

### DISCUSSION THREAD

**Discussion by Smiog** [razonyang/hugo-theme-bootstrap#356] [Q&A]
![image](https://user-images.githubusercontent.com/90970249/150782091-6552577a-263f-465e-9d6b-dce88ca81b59.png)

![image](https://user-images.githubusercontent.com/90970249/150782219-843caa39-e4e7-44ae-879f-999c07b83898.png)

# After a user submitted the form, where can I find it?

**Comment by razonyang** [comment]
The example site uses Getform as the backend. You need to change it to your own.

See also [Contact Form](https://hbs.razonyang.com/en/docs/layouts/contact-form/).
